tkhomich/03_html/Forms_2.html

```diff
@@ -0,0 +1,137 @@
+            <legend>Строка поиска</legend>
+            <input type="text" name="search" maxlength="10">
+            <input type="submit" value="Искать"><br>
+                <label><input type="radio" name="search_dist" value="site" checked>
+            на сайте</label>
+            <label><input type="radio" name="search_dist" value="google">
+            в Google</label><br>
+            <em></em>
+        </fieldset>
+        </form>
+    </td>
+    <td>
+        <h2>Простая форма поиска с вариантами:</h2>
+        <form method="get" action="http://sc.it-academy.by/fe/form.php">
+ 	    <fieldset>
+ 	    	<legend>Строка поиска</legend>
+ 		    <input type="text" name="search" maxlength="10" required autofocus><span>*</span>
+ 		    <input type="submit" value="Искать"><br>
+ 		    <label><input type="radio" name="search_dist" value="site">
+ 			на сайте</label>
+ 		    <label><input type="radio" name="search_dist" value="google" checked>
+ 			в google</label><br>
+ 	    </fieldset>
+        </form>
+    </td>
+    <tr>
+        <td>
+            <h2>Простая форма авторизации:</h2>
+            <form method="post" action="http://sc.it-academy.by/fe/form.php">
+ 	        <fieldset>
+ 	        	<legend>Авторизуйтесь</legend>
+                <label><input type="text" name="login" maxlength="20" required><span>*</span>Логин</label><br>
+                <br>
+                <label><input type="text" name="password" maxlength="20" required><span>*</span>Пароль</label><br>
+ 		        <label><input type="checkbox" name="stay_on_site" value="site">
+ 			Оставаться на сайте</label><br>
+                <input type="submit" name="signin" value="Войти"><br>
+ 	        </fieldset>
+            </form>
+        </td>
+        <td>
+            <h2>Опрос (голосование): </h2>
+            <form method="post" action="http://sc.it-academy.by/fe/form.php">
+	        <fieldset>
+		        <legend> Как вам?</legend>
+                <label><input type="radio" name="radio_group" value="value_01" id="Checkbox_Group1_0" checked>ничего</label><br>
+                <label><input type="radio" name="radio_group" value="value_02" id="Checkbox_Group2_0">неплохо</label><br>
+                <label><input type="radio" name="radio_group" value="value_03" id="Checkbox_Group3_0">хорошо</label><br>
+                <label><input type="radio" name="radio_group" value="value_04" id="Checkbox_Group4_0">супер</label><br>
+                <input type="hidden" name="date_vote" value="2013.12.31">
+                <input type="submit" value=Отправить>       
+	        </fieldset>
+            </form>
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <h2>Отзыв:</h2>
+            <br>
+            <form method="post" action="http://sc.it-academy.by/fe/form.php">
+                 <fieldset>
+                    <legend>Оставьте Ваш отзыв</legend>
+                    <label><input type="text" name="login" maxlength="20" required><span>*</span>Имя</label><br>
```

Comment:
this method (label implementation) we use if input has value of attribute type=radio or checkbox